checker/src/expression.rs

```diff
@@ -75,6 +75,12 @@ pub enum Expression {
         right: Box<AbstractDomain>,
     },
 
+    /// An expression that is the operand cast to the target_type. as
```

Comment:
eh?

Reply:
Looks a bit weird, but if you look at the surrounding context you'll see that each such comment mentions the operator symbol at the end.